src/dotnet-roslyn-tools/PRFinder/Formatters/DefaultFormatter.cs

```diff
@@ -6,8 +6,8 @@ namespace Microsoft.RoslynTools.PRFinder.Formatters;
 
 public class DefaultFormatter : IPRLogFormatter
 {
-    public virtual string FormatChangesHeader(string previous, string previousUrl, string current, string currentUrl)
-        => $"### Changes from [{previous}]({previousUrl}) to [{current}]({currentUrl}):";
+    public virtual string FormatChangesHeader(string start, string startUrl, string end, string endUrl)
```

Comment:
Probably not a huge deal, but should the header change to include the filter applied? Since it won't be showing all commits